daemon/cmd/daemon_main.go

```diff
@@ -1722,6 +1722,16 @@ func runDaemon() {
 	bootstrapStats.updateMetrics()
 	go d.launchHubble()
 
+	err = option.Config.StoreInFile(option.Config.RunDir)
+	if err != nil {
+		log.WithError(err).Error("Unable to store running configuration")
+	}
+
+	err = option.StoreViperInFile(option.Config.RunDir)
+	if err != nil {
+		log.WithError(err).Error("Unable to store running configuration")
+	}
```

Comment:
Is the difference between the two that one is runtime config and the other is what Cilium was configured with?

Reply:
yeah you can see it that way. some fields in `option.Config` are changed so it will be good to know what is Cilium actually running with, while `viper.Config` is exactly what is passed down to fiper.